UK/Data/Settings/Local/Edinburgh/Symbology.txt

```diff
@@ -97,20 +97,20 @@ Ground Network:taxiway:65280:3.5:0:3:7
 Ground Network:terminal taxiway:255:3.5:0:3:7
 Sector:line:21331:0.5:2:2:7
 Sector:msaw:32768:2.0:0:2:7
-Sector:active sector background:5519132:3.5:0:0:7
-Sector:inactive sector background:5519132:3.5:0:0:7
+Sector:active sector background:8747638:3.5:0:0:7
```

Comment:
I'm not sure these colours are reflective of the InNOVA displays... 
I'd be careful of just colour picking from what is possibly an over/under exposed image of the real system.
I'd like some input from the users of the real system to compare.